README.md

```diff
@@ -1 +1,52 @@
-# grid-contrib
\ No newline at end of file
+# grid-contrib
+
+The grid-contrib repository contains contributed software for
+[Hyperledger Grid](https://grid.hyperledger.org), a framework for building
+supply chain solutions for blockchain platforms.
+
+This repository is intended for community-contributed software, including
+example middleware tools and reference implementations for smart contracts
```

Comment:
middleware tools? feels like 'domain models' and 'smart contracts' are a better emphasis here. thoughts?

Reply:
@davececchi - How does "including example domain models and smart contracts" sound?  

Or would you keep the "reference implementations" piece?  I.e., "including example domain models and reference implementations for smart contracts"?

A third option would be to drop "example": "including domain models and {reference implementations of} smart contracts".

Reply:
Follow-up question: Is a domain model the same thing as a data mode"?  (The grid repo README uses the term "data model").